libraries/abstractions/common_io/test/test_iot_i2c.c

```diff
@@ -357,7 +360,7 @@ TEST( TEST_IOT_I2C, AFQP_IotI2CSetGetConfigurationFail )
         TEST_ASSERT_EQUAL( IOT_I2C_INVALID_VALUE, lRetVal );
 
         /* i2c ioctl with unsupported request */
-        lRetVal = iot_i2c_ioctl( xI2CHandle, testIotI2C_INVALID_IOCTL_INDEX, &xI2CConfig );
+        lRetVal = iot_i2c_ioctl( xI2CHandle, ( IotI2CIoctlRequest_t ) testIotI2C_INVALID_IOCTL_INDEX, &xI2CConfig );
```

Comment:
I notice that `testIotI2C_INVALID_IOCTL_INDEX` is defined as `UINT32_MAX`. Instead of explicitly downcasting it, should it be defined to be a lower value as `UINT8_MAX` instead or any other unsupported value?

Reply:
I think this is a better idea. Will do.